pyproject.toml

```diff
@@ -24,6 +24,28 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+[build-system]
+requires = ["setuptools>=42", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "perf-analyzer"
+dynamic = ["version"]
+description = "Triton Performance Analyzer"
+readme = "README.md"
+
+[tool.setuptools.dynamic]
+version = {attr = "perf_analyzer.__version__"}
+
+[tool.setuptools.packages.find]
+where = ["src"]
+
+[tool.setuptools.package-data]
+"perf_analyzer" = ["bin/perf_analyzer"]
+
+[project.scripts]
+perf_analyzer = "perf_analyzer.cli:main"
+
```

Comment:
This is just necessary stuff for making a PA wheel that is effectively a wrapper for the pre-existing binary.